rmw_fastrtps_shared_cpp/src/rmw_take.cpp

```diff
@@ -643,7 +643,8 @@ __rmw_return_loaned_message_from_subscription(
   std::unique_ptr<rmw_fastrtps_shared_cpp::LoanManager::Item> item;
   item = info->loan_manager_->erase_item(loaned_message);
   if (item != nullptr) {
-    if (!info->data_reader_->return_loan(item->data_seq, item->info_seq)) {
+    auto ret_code = info->data_reader_->return_loan(item->data_seq, item->info_seq);
+    if (eprosima::fastdds::dds::RETCODE_OK != ret_code) {
```

Comment:
There is no need to save the return value in a variable, right?
```suggestion
    if (eprosima::fastdds::dds::RETCODE_OK != info->data_reader_->return_loan(item->data_seq, item->info_seq)) {
```